### ISSUE TITLE
Downloaded Chapters Not Available After Update

### ISSUE THREAD

**Issue by AefRal** [Jays2Kings/tachiyomiJ2K#1745]
### Steps to reproduce

1. Have downloaded chapters.
2. Update from 1.7.3 to 1.7.4.
3. Go to read something only to see downloaded chapters missing.

### Expected behavior

After the update, my chapters remain.

### Actual behavior

The downloads location changed.

### TachiyomiJ2K version

1.7.4

### Android version

Android 10

### Device

Huawei P20 Pro CLT-L04

### Other details

Found the original download location of the chapters.
![Screenshot_20240122_135624_com huawei hidisk](https://github.com/Jays2Kings/tachiyomiJ2K/assets/68062495/8361959e-352f-49c0-b757-ecd35d1a0b6c)
Tried to change download location.
![Screenshot_20240122_135650_eu kanade tachiyomi j2k](https://github.com/Jays2Kings/tachiyomiJ2K/assets/68062495/38ef088b-542c-4ffd-94fd-98b2c3814de9)
Tapped Allow.
![Screenshot_20240122_135927_com android documentsui](https://github.com/Jays2Kings/tachiyomiJ2K/assets/68062495/965a6617-4dbe-46dd-9ba8-bf0d24436bea)
Yet still, they don't show up.
![Screenshot_20240122_140111_eu kanade tachiyomi j2k](https://github.com/Jays2Kings/tachiyomiJ2K/assets/68062495/d250903f-2a71-4b63-9388-5051a0c0c210)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.4](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by EffakT** [comment]
Guessing something got out of sync with the last update.
Ensure the download location is back to what it was before you changed it.

If you go to Settings > Advanced > Refresh the download cache.

Now go back to read something, are the downloads back?

**Comment by EffakT** [comment]
Hmm, okay. If you go to Extensions, are they all "Trusted"? If they have Trust next to them, you will need to trust each one.
Browse > Extensions.

It should either have Settings or Trust.

**Comment by AefRal** [comment]
Well that's new.
All my extensions in browse were gone, then I tapped the extension button and they were all there but had that Trust button on each. So I tapped it on certain ones and checked my library. Didn't work. Then I trusted all my extensions. And thought to refresh the download cache. That worked!

Thanks.

**Comment by EffakT** [comment]
Yeah, I think the Trust was added as part of 3rd party extensions. Sorry, I completely forgot I had to trust them before refreshing. At least its sorted now.

Probably leave the issue open since these stwpa shouldn't really be necessary, and should have either been documented or maybe the extensions installed pre update auto-trusted.

**Comment by AefRal** [comment]
Agreed. When getting the update I read the changes, there wasn't anything about trusting extensions. Don't normally read full changelog, but read it this time and saw nothing about this.
